packages/components/src/presentation-components/instance-filter-builder/PresentationInstanceFilter.ts

```diff
@@ -354,11 +354,22 @@ function parseGenericFilterRuleGroup(
 
 function parseGenericFilterRule(rule: GenericInstanceFilterRule, ctx: GenericFilterParsingContext): PresentationInstanceFilterCondition {
   const field = ctx.findField(rule.propertyName, rule.sourceAlias);
+  let value = rule.value ? rule.value.rawValue : undefined;
+
+  if (value && rule.operator === "like" && typeof value === "string") {
+    if (value.startsWith("%")) {
+      value = value.slice(1);
+    }
+
+    if (value.endsWith("%")) {
+      value = value.slice(0, -1);
+    }
```

Comment:
I think you can use regex.

```suggestion
    value = value.replace(/^%|%$/g, "");
```